app/mainAppWindow/index.js

```diff
@@ -252,6 +252,13 @@ function onBeforeRequestHandler(details, callback) {
 		logger.debug(`Forwarding '${details.url}' to '${imgUrl}'`);
 		callback({ redirectURL: imgUrl });
 	}
+	// Custom background replace for teams v2
+	else if (details.url.startsWith('https://statics.teams.cdn.office.net/evergreen-assets/backgroundimages/') && config.isCustomBackgroundEnabled) {
```

Comment:
As this implementation replaces Microsoft's default backgrounds, to use them it is necessary to disable custom backgrunds, with "isCustomBackgroundEnabled=false".
In the future we could make an improvement to allow both MSBackgrunds and customBackgrounds to be available.

Reply:
A bit confused here, but that logic will check if the url is the new evergreen and is the `isCustomBackgroundEnabled` true. Then it will replace the MS images. 

I think that is correct, as the isCustomBackgroundEnabled is also used in the main index.js https://github.com/IsmaelMartinez/teams-for-linux/blob/v1.4.31/app/index.js#L220 to load the config from a local server. Without the local server there is no way to get the images :D